### ISSUE TITLE
Can't render SVGs as Action icons

### ISSUE THREAD

**Issue by FezVrasta** [raycast/extensions#1094]
Raycast version: 1.30.2

## Description

Icons render all solid:

![CleanShot 2022-03-13 at 9 29 43@2x](https://user-images.githubusercontent.com/5382443/158051576-7c3961fb-b1e3-401a-8662-e3343b339412.png)

It should look like this:
<img width="58" alt="CleanShot 2022-03-13 at 9 31 21@2x" src="https://user-images.githubusercontent.com/5382443/158051589-1d257618-13a8-4038-b850-a0f2f3059da0.png">

SVG:

```
<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="1" stroke-linecap="round" stroke-linejoin="round" class="feather feather-play-circle">
  <circle cx="12" cy="12" r="10"></circle>
  <polygon points="10 8 16 12 10 16 10 8"></polygon>
</svg>
```

## Steps To Reproduce

1. <Action icon={{ source: 'play-circle.svg' }} />

**Comment by fe9lix** [comment]
SVGs should generally work in an ActionPanel, see:

<img width="365" alt="CleanShot 2022-03-23 at 09 13 48@2x" src="https://user-images.githubusercontent.com/213498/159653508-a741ed82-2cbb-4eff-aced-0657ad902b9c.png">

The SVG is:
```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 100 100">
  <rect fill="#D22" x="0" y="3" height="95" width="95" rx="15"/>
  <circle cx="18" cy="81" r="9" fill="#FFF"/>
  <path d="M48,84s0-33-33-33 M75,84s0-60-60-60"
    stroke-linecap="round" stroke-width="15" stroke="#FFF" fill="none"/>
</svg>
```

It might be something specific with this icon that our SVG parser doesn't like.
Closing this issue for now.

**Comment by FezVrasta** [comment]
Why are you closing it ? Is valid svg and comes from a popular icon set

**Comment by fe9lix** [comment]
We'll take another look but I can't promise that we're able to fix this soon. Meanwhile, if you find out what might be "special" about this particular SVG, let us know.

**Comment by FezVrasta** [comment]
I wonder if it's because the `stroke-*` attributes are attached to the `svg` element directly.

**Comment by FezVrasta** [comment]
Ok I can confirm it seems to be because of the `stroke-*` attributes on `svg`, tbh I never saw this way of applying attributes to the child elements, it looks like anything applied to the `svg` attribute is automatically applied to any child element too?

**Comment by nhojb** [comment]
This will be fixed in the next version (1.36) when it is released.
